### ISSUE TITLE
Human-readable output of activity-date rules are unclear (and don’t mention the condition)

### ISSUE THREAD

**Issue by andylolz** [IATI/IATI-Rulesets#227]
On the [activity-date page](http://reference.iatistandard.org/203/activity-standard/iati-activities/iati-activity/activity-date/) of the reference site:

![Screenshot 2019-09-20 at 09 53 46](https://user-images.githubusercontent.com/464193/65313639-9c8e8100-db8c-11e9-9a78-7bdf92e69617.png)

I’m also unsure where these rules come from. They don’t seem to be mentioned in the new (or old) guidance here:
http://reference.iatistandard.org/203/activity-standard/overview/dates/

**Comment by andylolz** [comment]
This relates to #156.

**Comment by samuele-mattiuzzo** [comment]
Thank you for flagging, we're going to discuss this with the BAs and get back to you as soon as we can. FWIW: in [Data4Development's IATI Ruleset repository](https://github.com/data4development/IATI-Rulesets) we're working to sort all these ambiguities out and once the repo is ready and stable, it'll replace this one (and the content on the Reference site will use it as well). We just don't have an ETA for that yet (shipping the whole validator is higher priority), so we'll carry on with maintenance on this

**Comment by andylolz** [comment]
Great – sounds good.

I’ve sent a PR (#228) in case it’s useful.

**Comment by andylolz** [comment]
This issue is no longer relevant as these rules have been removed. Closing
